timescaledb/how-to-guides/connecting/azure-data-studio.md

```diff
@@ -54,10 +54,18 @@ services.
 1.  In the `SERVERS` page, click `New Connection`.
 1.  In the `Connection Details` dialog, navigate to `Connection type`, and select `PostgreSQL`.
     Fill in the fields using the server name, database, user name, port, and password for your
-    TimescaleDB Cloud service. For more information about these settings, see 
+    TimescaleDB Cloud service. For more information about these settings, see
     [the connection details section][connection-details].
 1.  Click `Advanced`.
 1.  In the `Port` field, type the port number, and click `OK`.
+
+    <highlight type="note">
+    If you configured your Timescale Cloud service to connect using
+    [SSL mode] [ssl-mode], then set `SSL mode` to `Verify-Full`, and in the
```

Comment:
The SSL mode link doesn't seem to work. (it might be just the unnecessary space in-between or because it's a note block?)

Reply:
Yes!!! the note block, thanks let me fix that.